src/tutorials.js

```diff
@@ -5,6 +5,14 @@ const moment = require('moment');
 const axiosPath = require('./axiosPath');
 
 const tutorials = [
+  {
+    title: 'Mongoose find() Certain Fields',
+    raw: './tutorials/mongoose/filter-find.md',
+    url: '/tutorials/mongoose/filter-find',
```

Comment:
Keep the URL slug and filename as "find-certain-fields" and "find-certain-fields.md" to be consistent with the keyword we're targeting.